src/openrct2/core/ChecksumStream.cpp

```diff
@@ -11,6 +11,7 @@
 
 #include "Endianness.h"
 
+#include <array>
```

Comment:
This is unnecessary. std::array is already included in the header, and here it's only used for a function definition that comes from the same header.